tests/Jellyfin.Api.Tests/ParseNetworkTests.cs

```diff
@@ -37,28 +37,28 @@ public void TestNetworks(bool ip4, bool ip6, string hostList, string match)
                 EnableIPV6 = ip6
             };
 
-            var result = match + ',';
+            var result = match + ",";
```

Comment:
Why switch these back to double quotes?

Reply:
Otherwise this char gets boxed for the concatenation.